csharp/GildedRoseTest.cs

```diff
@@ -12,7 +12,130 @@ public void foo()
             IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
             GildedRose app = new GildedRose(Items);
             app.UpdateQuality();
-            Assert.AreEqual("fixme", Items[0].Name);
+            Assert.AreEqual("foo", Items[0].Name);
+        }
+
+        [Test]
+        public void AfterSellByDate_QualityDegradesTwiceAsFast()
```

Comment:
It would be useful to split the tests into `//given`, `//when`, `//then` blocks to make them easier to read